### DISCUSSION TITLE
[Question] Can murxla transform API input to SMT-LIB?

### DISCUSSION THREAD

**Discussion by merlinsun** [murxla/murxla#5] [General]
Hi,

I have a question regarding the capabilities of murxla. I noticed that some bug reports submitted by murxla are in SMT-LIB format, which led me to wonder if murxla is capable of transforming API input to SMT-LIB format. If that is the case, could you kindly advise me on how to utilize this function?

Additionally, I would like to know if murxla can transform SMT-LIB format to API input. I would appreciate any information you can provide on this matter.

Thank you for your help and guidance.

Best regards

**Comment by mpreiner** [comment]
You can test solvers via the SMT2 interface as described here: https://murxla.github.io/docs/user_guide.html#testing-via-the-smt-libv2-interface

Murxla does not support translating SMT2 to API input.

**Reply by merlinsun** [reply]
Thanks for your help!
